airbyte-integrations/connectors/source-mssql/src/main/java/io/airbyte/integrations/source/mssql/MssqlCdcHelper.java

```diff
@@ -94,8 +94,6 @@ static Properties getDebeziumProperties(final JdbcDatabase database, final Confi
             ? HEARTBEAT_INTERVAL_IN_TESTS
             : HEARTBEAT_INTERVAL;
     props.setProperty("heartbeat.interval.ms", Long.toString(heartbeatInterval.toMillis()));
-    // TODO: enable heartbeats in MS SQL Server.
-    props.setProperty("heartbeat.interval.ms", "0");
```

Comment:
Side not : Opened this issue https://github.com/airbytehq/airbyte/issues/34812 to just get rid of heartbeats altogether